### DISCUSSION TITLE
Hide Audio Devices That Are Not Commonly Used

### DISCUSSION THREAD

**Discussion by D-Arora** [File-New-Project/EarTrumpet#1076] [Ideas]
It would be great to be able to permanently hide certain audio devices that get used infrequently (Like RTX Voice etc.) in an extended menu just to minimise clutter.

**Comment by SeeGeeArtist** [comment]
It seems a no-brainer to be able to curate the list. I have a few virtual cables I never use and steam vr adds a bunch of devices for routing audio to my headset that I never interact with.
